README.md

```diff
@@ -1,70 +1,53 @@
-# Getting Started with Create React App
+![](https://img.shields.io/badge/Microverse-blueviolet)
```

Comment:
- [ ] Great work adding a readme to your project. However, It will be great to use the new and Improved template by Microverse. This is an important requirement for all projects before they can be approved. To check the new readme, kindly follow this  [link](https://github.com/microverseinc/readme-template)